src/app/workflow/info-tab/info-tab.component.html

```diff
@@ -384,6 +384,11 @@
             </span>
           </div>
         </div>
+        <div *ngIf="selectedVersion?.frozen && hasHttpImports" class="pt-3">
```

Comment:
What does this look like in terms of screenshot?
Thinking it would be nice to link somewhere for more info. We don't have somewhere great for now, but perhaps link to https://docs.dockstore.org/en/stable/advanced-topics/snapshot-and-doi.html?highlight=imports#what-is-included-in-a-snapshot